modules/op-tkn-pipelines-compatibility-support-matrix.adoc

```diff
@@ -19,7 +19,7 @@ GA:: General Availability
 
 | Operator | Pipelines | Triggers | CLI | Chains | Hub | {pac} | Results | Manual Approval Gate | |
 
-|1.15 | 0.59.x | 0.27.x | 0.37.x | 0.20.x (GA) | 1.17.x (TP) | 0.27.x (GA) | 0.10.x (TP) | 0.2.x (TP) | 4.14, 4.15, 4.16 | GA
+|1.15 | 0.59.x | 0.27.x | 0.37.x | 0.20.x (GA) | 1.17.x (TP) | 0.27.x (GA) | 0.10.x (TP) | 0.2.x (TP) | 4.14, 4.15, 4.16 (*) | GA
```

Comment:
here also it should be 4.12, 4.14 etc...

Reply:
discussed on Slack - this line is for 1.15 so likely to be read as 1.15.0